### ISSUE TITLE
Compile issue Win 10

### ISSUE THREAD

**Issue by SteveRussell33** [Rcomian/jsfx-vcv#2]
g++  -Wsuggest-override -std=c++11 -DSLUG=jsfx -fPIC -I../../include -I../../dep/include -DVERSION=0.6.0 -MMD -MP -g -O3 -march=nocona -ffast-math -fno-finite-math-only -Wall -Wextra -Wno-unused-parameter -DARCH_WIN -D_USE_MATH_DEFINES -c -o build/src/JSFXModule.cpp.o src/JSFXModule.cpp
src/JSFXModule.cpp: In member function 'virtual void JSFXModule::onSampleRateChange()':
src/JSFXModule.cpp:30:3: error: 'uint' was not declared in this scope
   uint sliderid = 0;
   ^~~~
src/JSFXModule.cpp:30:3: note: suggested alternative: 'u_int'
   uint sliderid = 0;
   ^~~~
   u_int
src/JSFXModule.cpp:37:29: error: 'sliderid' was not declared in this scope
         _jsusfx->moveSlider(sliderid, slider.max);
                             ^~~~~~~~
src/JSFXModule.cpp:37:29: note: suggested alternative: 'slider'
         _jsusfx->moveSlider(sliderid, slider.max);
                             ^~~~~~~~
                             slider
src/JSFXModule.cpp:39:29: error: 'sliderid' was not declared in this scope
         _jsusfx->moveSlider(sliderid, slider.min);
                             ^~~~~~~~
src/JSFXModule.cpp:39:29: note: suggested alternative: 'slider'
         _jsusfx->moveSlider(sliderid, slider.min);
                             ^~~~~~~~
                             slider
src/JSFXModule.cpp:41:27: error: 'sliderid' was not declared in this scope
       _jsusfx->moveSlider(sliderid, current);
                           ^~~~~~~~
src/JSFXModule.cpp:41:27: note: suggested alternative: 'slider'
       _jsusfx->moveSlider(sliderid, current);
                           ^~~~~~~~
                           slider
src/JSFXModule.cpp:43:5: error: 'sliderid' was not declared in this scope
     sliderid += 1;
     ^~~~~~~~
src/JSFXModule.cpp:43:5: note: suggested alternative: 'slider'
     sliderid += 1;
     ^~~~~~~~
     slider
src/JSFXModule.cpp: In member function 'virtual void JSFXModule::step()':
src/JSFXModule.cpp:78:5: error: 'uint' was not declared in this scope
     uint sliderid = 0;
     ^~~~
src/JSFXModule.cpp:78:5: note: suggested alternative: 'u_int'
     uint sliderid = 0;
     ^~~~
     u_int
src/JSFXModule.cpp:79:9: error: expected ';' before 'paramid'
     uint paramid = 0;
         ^~~~~~~~
         ;
src/JSFXModule.cpp:82:34: error: 'paramid' was not declared in this scope
         auto paramvalue = params[paramid].value * slider.inc;
                                  ^~~~~~~
src/JSFXModule.cpp:82:34: note: suggested alternative: 'params'
         auto paramvalue = params[paramid].value * slider.inc;
                                  ^~~~~~~
                                  params
src/JSFXModule.cpp:94:29: error: 'sliderid' was not declared in this scope
         _jsusfx->moveSlider(sliderid, paramvalue);
                             ^~~~~~~~
src/JSFXModule.cpp:94:29: note: suggested alternative: 'slider'
         _jsusfx->moveSlider(sliderid, paramvalue);
                             ^~~~~~~~
                             slider
src/JSFXModule.cpp:97:7: error: 'sliderid' was not declared in this scope
       sliderid += 1;
       ^~~~~~~~
src/JSFXModule.cpp:97:7: note: suggested alternative: 'slider'
       sliderid += 1;
       ^~~~~~~~
       slider
make: *** [../../compile.mk:65: build/src/JSFXModule.cpp.o] Error 1

Building with Rack SDK 0.6.2 Win 10
jsusfx-0.4.0b1b is added in dep/

**Comment by Rcomian** [comment]
yeah tell me about it, I'm really struggling to get a Windows build, it's why I haven't released it.

if you find a way to resolve, I'll be absolutely delighted
